### ISSUE TITLE
[Bug] CollectionView with ItemsLayout=VerticalGrid crashes on Android

### ISSUE THREAD

**Issue by vskotatko** [xamarin/Xamarin.Forms#12920]
### Description
According to this page: [Xamarin.Forms CollectionView Layout](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/collectionview/layout) the CollectionView ItemsLayout property accepst four values, as shown in the examples on the page.

Indeed, using ItemsLayout="VerticalGrid, 2" works when running on the UWP. However, it crashes with an error on Android: xamarin System.InvalidOperationException: 'Cannot convert “VerticalGrid, 2” into Xamarin.Forms.IItemsLayout'.

It doesn't take much XAML to reproduce:

```
    <CollectionView
      x:Name="DetailGrid"
      ItemsLayout="VerticalGrid, 2" >

      <CollectionView.ItemTemplate>
        <DataTemplate>
          <Label Text="asdf" />
        </DataTemplate>
      </CollectionView.ItemTemplate>

    </CollectionView>

```

### Steps to Reproduce

1. 
2. 
3. 

### Environment

Microsoft Visual Studio Community 2019
Version 16.8.1
VisualStudio.16.Release/16.8.1+30711.63
Microsoft .NET Framework
Version 4.8.03752

Installed Version: Community

Visual C++ 2019   00435-60000-00000-AA332
Azure App Service Tools v3.0.0   16.8.550.19892
C# Tools   3.8.0-5.20519.18+4c195c3ac1974edcefa76774d7a59a2350ec55fa
Common Azure Tools   1.10
Extensibility Message Bus   1.2.6 (master@34d6af2)
GitHub.VisualStudio   2.11.106.19330
IntelliCode Extension   1.0
Microsoft JVM Debugger   1.0
Microsoft MI-Based Debugger   1.0
Microsoft Visual C++ Wizards   1.0
Microsoft Visual Studio VC Package   1.0
Mono Debugging for Visual Studio   16.8.43 (00471f8)
NuGet Package Manager   5.8.0
ProjectServicesPackage Extension   1.0
Visual Basic Tools   3.8.0-5.20519.18+4c195c3ac1974edcefa76774d7a59a2350ec55fa
Visual F# Tools   16.8.0-beta.20507.4+da6be68280c89131cdba2045525b80890401defd
Visual Studio Code Debug Adapter Host Package   1.0
VisualStudio.DeviceLog   1.0
VisualStudio.Foo   1.0
VisualStudio.Mac   1.0
Xamarin   16.8.000.255 (d16-8@d002176)
Xamarin Designer   16.8.0.507 (remotes/origin/d16-8@e87b24884)
Xamarin Templates   16.8.112 (86385a3)
Xamarin.Android SDK   11.1.0.17 (d16-8/c0e2b8e)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: be2226b
    Java.Interop: xamarin/java.interop/d16-8@79d9533
    ProGuard: Guardsquare/proguard/proguard6.2.2@ebe9000
    SQLite: xamarin/sqlite/3.32.1@1a3276b
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-8@2fb1cbc
Xamarin.iOS and Xamarin.Mac SDK   14.4.1.3 (e30c41de3)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

### Workaround

I discovered that the following syntax works around this problem:

    <CollectionView
      x:Name="DetailGrid"
      >

      <CollectionView.ItemsLayout>
        <GridItemsLayout Orientation="Vertical" Span="2" />
      </CollectionView.ItemsLayout>
      ...
    </CollectionView>

**Comment by jsuarezruiz** [comment]
I have created a sample using Xamarin.Forms 4.8 and two CollectionView's using:

- `ItemsLayout="HorizontalGrid, 2"`
- `ItemsLayout="VerticalGrid, 2"`

[Issue12920.zip](https://github.com/xamarin/Xamarin.Forms/files/5572878/Issue12920.zip)

But, cannot reproduce the issue on Android:
<img width="1692" alt="Captura de pantalla 2020-11-20 a las 11 27 05" src="https://user-images.githubusercontent.com/6755973/99789879-90e97200-2b23-11eb-9084-7e5d05d068c6.png">

Could you attach a sample where reproduce the issue or take a look to the attached sample?

**Comment by vskotatko** [comment]
Hi Javier,

It appears to be fixed in the latest Xamarin.Forms update (4.8.0.1687). 
I'm new to Xamarin.Forms and didn't realize that after updating packages 
through nuget, one also has to update them for each individual project. 
So, even though I had updated my environment to all the latest versions, 
apparently I was still building with some older versions. (This behavior 
isn't obvious to a new user and I didn't see it mentioned in any startup 
docs, though I easily could have missed it.)

I decided to leave the bug open for a bit just in case it reoccurred for 
some reason. But I think it's safe to close it. Since it's unlikely that 
the bug "fixed" itself, it would be nice to verify that someone actually 
fixed this in a recent update.

Thanks for your time.

Peri

<< Annoyed by leaf blowers ? https://quietcleanseattle.org/ >>

------ Original Message ------
